tools/emptyDropsWrapper/test/test_emptyDropsWrapper.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+## File: test_emptyDropsWrapper.sh
+## Author: Nikolas Barkas
+## Date: January 3, 2019
+## Description: Unit test for emptyDropsWrapper script
+
+## Download some sample data from 10X
+printf "Downloading pbmc4k data..."
+wget -q http://cf.10xgenomics.com/samples/cell-exp/2.1.0/pbmc4k/pbmc4k_raw_gene_bc_matrices.tar.gz
+printf "done\n"
+
+## Decompress dataset
+printf "Decompressing archive..."
+tar xzf pbmc4k_raw_gene_bc_matrices.tar.gz
+printf "done\n"
+
+## Prepare RDS files for reading from the the 10X matrix
+printf "Converting mtx to rds..."
+./prepRDS.R
+printf "done\n"
+
+## Run empty drops
+printf "Running emptyDrops..."
+../emptyDropsWrapper.R -i pbmc4k.rds -o pbmc4k_emptyDrops.csv
+printf "done\n"
+
+## Check the output md5 checksum
+## Note that empty drops is based on MC simulation and therefore
```

Comment:
What is an MC simulation?

Reply:
I guess it's referring to `Monte Carlo Simulation`? +1 for making it explicit!

Reply:
Yes! Please write out abbreviations that are less common at least in the first place it is used for better readability 📖

Reply:
done